package.json

```diff
@@ -121,23 +122,7 @@
   },
   "homepage": "https://github.com/SVG-Edit/svgedit#readme",
   "browserslist": [
-    "cover 100%",
-    "last 3 Chrome versions",
-    "last 3 Firefox versions",
-    "last 3 Edge versions",
-    "last 3 Safari versions",
-    "last 3 ios_saf versions",
-    "last 3 Android versions",
-    "last 3 Opera versions",
-    "last 3 IE versions",
-    "last 3 op_mini versions",
-    "last 3 bb versions",
-    "last 3 op_mob versions",
-    "last 3 and_chr versions",
-    "last 3 and_ff versions",
-    "last 3 ie_mob versions",
-    "last 3 and_uc versions",
-    "last 3 Samsung versions"
+    "> 2%"
   ],
```

Comment:
I know we talked about such a change, but with a `browserslist` change (which impacts build files), this should definitely be in the PR message too.

I'm guessing the choice was to minimize the build size? I'd personally prefer just dropping the cover 100% and keeping the existing version targets--though potentially changing them as needed if any old versions were adding a particular large amount of bloat--as that would convey more precisely which versions were simply too problematic to support--rather than forcing us to arbitrarily doom 2% of users even into the future.

If we do make a change, we might also see whether we can remove some items from `settings.polyfills` as those were used to suppress linting errors about older versions not supporting certain globals. (The easiest way to find out would probably be to remove those `settings`--including those in `overrides`, and see which are still necessary. Such an exercise might even help point us to the versions we want to drop--e.g., if a browser doesn't support `Error` or `console`, that is a pretty crusty browser!) :-)

Reply:
I like your changes to exclude certain browser versions instead, thanks.

However, I will mention here in connection with `browserlist` that `compat/compat` should not be a mere warning. That rule indicates what browser versions we are supporting, and if we are breaking that rule, our code will stop working for projects that expect it to work. We either need to:

1. Suppress the warning by adding to `settings.polyfills` in `.eslintrc.js` but don't provide our own polyfill. This is what we have been doing, but if, as with this PR, we are settling on some specific browsers, I think we really should also add mention on the README that in order to support such-and-such a browser, the following polyfills will be needed, along with links to at least one popular polyfill for any missing feature so projects are not left in the dark.
2. Provide our own internal polyfill (and suppress the warning by adding to `settings.polyfills` in `.eslintrc.js`).
3. Update `browserslist` to remove support for versions that cause undue breakage--with at least a reference in `CHANGES.md` (and it should be part of a breaking release change, though in this case, our next one is already going to be breaking anyways). Whenever we do this, I hope it is a principled decision based on the browser having low market share AND causing undue bloat for polyfilling its ESLint features. Otherwise, it isn't hard to specify `settings.polyfills` so projects wishing to support very old versions can still do so.

In any case, we should summarize our `browserslist` on the README, under a new "Requirements" section, as users shouldn't need to dig into our `package.json` to know what versions are expected to work (and what polyfills are needed to even support our stated supported versions).

Plan to reply below re: other eslint-related changes.

Reply:
I need a little more time to come with a solution which will likely be a combination of babel and rollup. I'm thinking about a build for older browsers with the necessary polyfills and another lighter/faster build for modern browsers. Indeed, it should be documented the requirement for each builds.

Reply:
I think it's already a lot of changes in this PR so for now, I don't want to review/change the build but we can do this as the next PR.